### PR TITLE
Tier1 tests added

### PR DESCRIPTION
**Pull request by ibragins**
Signed-off-by: Igor Braginsky <ibragins@redhat.com>

### CODE REVIEW

File: pkg/qe-tests/cypress/integration/models/plan.ts [hunk 1]
```diff
@@ -573,7 +575,7 @@ export class Plan {
       this.filterVm(duplicatePlanData);
       this.selectVm(duplicatePlanData);
       this.unSelectVm(vmListRemove);
-      this.networkMappingStep(duplicatePlanData);
```
Comment (yaacov) [inline pkg/qe-tests/cypress/integration/models/plan.ts:576]: @ibragins sorry for the layman question, why do we skip the netwok step?

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_config_vmware.ts [hunk 2]
```diff
@@ -158,15 +158,14 @@ export const vmwareTier0Plan_nfs_cold: PlanData = {
...
   warmMigration: false,
   preHook: preHookData,
   postHook: postHookData,
 };
 
-//Defining vmware cold migration plan for ceph-rbd file system
-export const vmwareTier1Plan_ceph_cold: PlanData = {
-  name: `vmware-tier0-ceph-${vmwareProviderUser.name}-cold`,
+export const vmwareTier1Plan_ceph_cold_duplicate: PlanData = {
+  name: `a-vmware-tier0-ceph-${vmwareProviderUser.name}-cold-duplicate`,
```
Comment (yaacov) [inline pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_config_vmware.ts:168]:
```suggestion
  name: `a-vmware-tier1-ceph-${vmwareProviderUser.name}-cold-duplicate`,
```
it this `tier0` name on `tier1` test intentional ?

**Review by yaacov — APPROVED** [review]
have some nitpicks, o/w ok by me
@mturley please review

File: pkg/qe-tests/cypress/integration/models/plan.ts (hunk 1 above)
Comment (ibragins) [reply pkg/qe-tests/cypress/integration/models/plan.ts:576]: Cause there is no change expected. Also if I will expect any change in network mapping - I will need to control some how if this a new plan creation or editing/duplicating it which will cause serious refactoring. I was asked not to put too much effort in this.

File: pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_config_vmware.ts (hunk 2 above)
Comment (mturley) [reply pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_config_vmware.ts:168]: It probably makes sense to commit this rename, yes?

**Review by mturley — COMMENTED** [review]
LGTM aside from the name change mentioned above.

**Review by mturley — APPROVED** [review]

**Pull request merged**